neofetch

```diff
@@ -926,14 +926,21 @@ get_wm_theme() {
         ;;
 
         "Quartz Compositor")
-            wm_theme="$(PlistBuddy -c "Print AppleAquaColorVariant" \
-                        "${HOME}/Library/Preferences/.GlobalPreferences.plist")"
+            global_preferences="${HOME}/Library/Preferences/.GlobalPreferences.plist"
+            wm_theme="$(PlistBuddy -c "Print AppleInterfaceStyle" "$global_preferences")"
+            wm_theme_color="$(PlistBuddy -c "Print AppleAquaColorVariant" "$global_preferences")"
 
-            if [[ -z "$wm_theme" ]] || ((wm_theme == 1)); then
-                wm_theme="Blue"
+            if [[ -z "$wm_theme" ]]; then
```

Comment:
```sh
[[ -z "$wm_theme" ]] && em_theme="Blue"
```

Reply:
Not sure what you mean.  `AppleInterfaceStyle` is either `Dark` or missing/empty, and can be used with both Blue and Graphite.

Reply:
What I mean is there's no point in having `if; then fi` when there's only one statement inside the if. It's just a shorter way of writing the if.

Reply:
Gotcha, I've changed it